src/extract.ts

```diff
@@ -171,6 +171,70 @@ type JuliaBenchmarkGroup = [
 
 type JuliaBenchmarkJson = [object, JuliaBenchmarkGroup[]];
 
+export interface BenchmarkDotnetBenchmark {
+    DisplayInfo: string;
+    Namespace: string;
+    Type: string;
+    Method: string;
+    MethodTitle: string;
+    Parameters: string;
+    FullName: string;
+    Statistics: {
+        StandardDeviation: number;
+        Mean: number; // there are more, but this is all we care about now.
+    };
+    Memory:
+        | {
+              Gen0Collections: number;
+              Gen1Collections: number;
+              Gen2Collections: number;
+              TotalOperations: number;
+              BytesAllocatedPerOperation: number;
+          }
+        | undefined;
+    Measurements: [
+        {
+            IterationMode: 'Overhead' | 'Workload' | 'Unknown';
+            IterationStage: 'Jitting' | 'Pilot' | 'Warmup' | 'Actual' | 'Result' | 'Unknown';
+            LaunchIndex: number;
+            IterationIndex: number;
+            Operations: number;
+            Nanoseconds: number;
+        },
+    ];
+    Metrics: [
+        {
+            Value: number;
+            Descriptor: {
+                Id: string;
+                DisplayName: string;
+                Legend: string;
+                NumberFormat: string;
+                UnitType: 0 | 1 | 2; // unit type
+                Unit: string;
+                TheGreaterTheBetter: boolean;
+            };
+        },
+    ];
```

Comment:
Do we even care about this piece of the object? If not maybe let's remove this from the type